src/transformers/models/electra/modeling_electra.py

```diff
@@ -972,9 +972,11 @@ def __init__(self, config):
     @add_start_docstrings_to_model_forward(ELECTRA_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
         processor_class=_TOKENIZER_FOR_DOC,
-        checkpoint=_CHECKPOINT_FOR_DOC,
+        checkpoint="bhadresh-savani/electra-base-emotion",
```

Comment:
Hi @kumar-abhishek, 

We need to use task-specific checkpoints so the same checkpoint can't be used in different tasks since the model head will be different. I am following these [pull request ](https://github.com/huggingface/transformers/pull/16363) changes.

If we follow your suggestion, we need to make changes in all the earlier PR for the sprint since they are also following the same.

Please let me know your view on this

Reply:
I see, I am fine with this then.